.github/workflows/build.yaml

```diff
@@ -29,6 +29,11 @@ on:
         required: true
         type: boolean
         default: false
+      lsp4ijPrNumber:
+        description: 'If you checked this, please make sure to enter only the LSP4IJ PR number below'
+        required: true
+        type: boolean
+        default: false
       refLsp4ij:
         description: 'Reference/branch for Lsp4ij checkout'
```

Comment:
```suggestion
        description: 'Reference/branch/PR for LSP4IJ checkout'
```

Reply:
I modified the code based on your suggestion